File: main_app/static/js/urls.js
// ========== LIST ========== //
// ingrediente
window.listIngrediente = 'http://localhost:8000/api/ingredientes/list/';

// unidade_medida
window.listUnidadeMedida = 'http://localhost:8000/api/unidadesmedida/list/';

// aula
window.listAula = 'http://localhost:8000/api/aulas/list/';

// receita
window.listReceita = 'http://localhost:8000/api/receitas/list/';

// associativa aula_receita
window.listAulaReceita = 'http://localhost:8000/api/aula_receita/list/';

// associativa receita_ingrediente
window.listReceitaIngrediente = 'http://localhost:8000/api/receita_ingrediente/list/';

// ========== LIST fim ========== //

function load_url() {
    if (typeof idData === 'undefined') {
        window.idData = 0;
    }
    // ========== INGREDIENTE ========== //
    window.createIngrediente = 'http://localhost:8000/api/ingredientes/create/';
    window.updateIngrediente = 'http://localhost:8000/api/ingredientes/edit/' + idData + '';
    window.deleteIngrediente = 'http://localhost:8000/api/ingredientes/delete/' + idData + '';

    // ========== AULAS ========== //
    window.createAula = 'http://localhost:8000/api/aulas/create/';
    window.updateAula = 'http://localhost:8000/api/aulas/edit/' + idData + '';
    window.deleteAula = 'http://localhost:8000/api/aulas/delete/' + idData + '';

    // ========== AULA RECEITA ========== //
    window.createAulaReceita = 'http://localhost:8000/api/aula_receita/create/';
    window.deleteAulaReceita = 'http://localhost:8000/api/aula_receita/delete/' + idData + '';

    // ========== RECEITA INGREDIENTE ========== //
}

// JSONs ARMAZENDADOS para serem utilizados em todos os lugares
window.jsonUnidade;
window.jsonIngrediente;
window.jsonAula;
window.jsonReceita;
window.jsonAulaReceita;
window.jsonReceitaIngrediente;

File: relacionamentos/urls.py
from django.conf.urls import url
from .views import CreateAula, ListAula, EditAula, DeleteAula
from .views import CreateIngrediente, ListIngrediente, EditIngrediente, DeleteIngrediente
from .views import CreateReceita, ListReceita, EditReceita, DeleteReceita

from .views import CreateAulaReceita, ListAulaReceita,  DeleteAulaReceita
from .views import CreateReceitaIngrediente, ListReceitaIngrediente, DeleteReceitaIngrediente

urlpatterns = [
    url(r'^aulas/create/$', CreateAula.as_view(), name='create'),
    url(r'^aulas/list/$', ListAula.as_view(), name='list'),
    url(r'^aulas/edit/(?P<id>[0-9]+)$', EditAula.as_view(), name='edit'),
    url(r'^aulas/delete/(?P<id>[0-9]+)$', DeleteAula.as_view(), name='delete'),

    url(r'^ingredientes/create/$', CreateIngrediente.as_view(), name='create'),
    url(r'^ingredientes/list/$', ListIngrediente.as_view(), name='list'),
    url(r'^ingredientes/edit/(?P<id>[0-9]+)$', EditIngrediente.as_view(), name='edit'),
    url(r'^ingredientes/delete/(?P<id>[0-9]+)$', DeleteIngrediente.as_view(), name='delete'),

    url(r'^receitas/create/$', CreateReceita.as_view(), name='create'),
    url(r'^receitas/list/$', ListReceita.as_view(), name='list'),
    url(r'^receitas/edit/(?P<id>[0-9]+)$', EditReceita.as_view(), name='edit'),
    url(r'^receitas/delete/(?P<id>[0-9]+)$', DeleteReceita.as_view(), name='delete'),


    url(r'^aula_receita/create/$', CreateAulaReceita.as_view(), name='create'),
    url(r'^aula_receita/list/$', ListAulaReceita.as_view(), name='list'),
    url(r'^aula_receita/delete/(?P<id>[0-9]+)$', DeleteAulaReceita.as_view(), name='delete'),


    url(r'^receita_ingrediente/create/$', CreateReceitaIngrediente.as_view(), name='create'),
    url(r'^receita_ingrediente/list/$', ListReceitaIngrediente.as_view(), name='list'),
    url(r'^receita_ingrediente/delete/(?P<id>[0-9]+)$', DeleteReceitaIngrediente.as_view(), name='delete'),
     

]